modules/Login/containers/LoginContainer.tsx

```diff
@@ -10,20 +14,22 @@ import { ApiError } from "./LoginContainer.types";
 
 const LoginContainer = () => {
   const [login] = useLoginMutation();
+  const dispatch = useDispatch();
+  const router = useRouter();
 
   const handleSubmit = async (data: ILoginFormValues) => {
     try {
       const result = await login(data).unwrap();
-
+      dispatch(setUser(result.user));
+      dispatch(setToken(result.token));
       localStorage.setItem("authToken", result.token);
 
       notifications.show({
         title: "Success",
         message: "Login successful!",
         color: "blue",
       });
-
-      // TODO Redirect based on user type
+      router.push(`/dashboard/${result.user.userType}`);
     } catch (error) {
       console.error("Login failed:", error);
```

Comment:
Please add a toast message to show errors instead of console logging.

Reply:
Create a seperate ticket to add toast notifications and update all instances where you console logged in catch block.